src/main/webapp/create-live-stream.html

```diff
@@ -0,0 +1,31 @@
+<!doctype html>
```

Comment:
This page would still have the standard "Shef" headers and footers?

Reply:
Yes